catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerTest.java

```diff
@@ -78,66 +60,30 @@
 
     public static MockedAdminPoller poller;
 
-    private static final String REGISTRY_ID = "registry-id";
-
-    private static final String PUBLISHED_LOCATIONS = "published-locations";
-
-    @Mock
-    private CatalogFramework catalogFramework;
-
-    @Mock
-    private CatalogStore catalogStore1;
-
-    @Mock
-    private CatalogStore catalogStore2;
-
-    @Mock
-    private CatalogStore catalogStore3;
-
-    private FilterBuilder filterBuilder;
-
-    private Map<String, CatalogStore> catalogStoreMap;
-
-    private ArrayList<String> publishedPlaces;
-
-    private ArrayList<String> destinations;
-
     private List<Result> results;
 
     private Metacard metacard1;
 
     @Before
     public void setup() {
-        catalogFramework = mock(CatalogFramework.class);
-        catalogStoreMap = new HashMap<>();
-
-        catalogStore1 = mock(CatalogStore.class);
-        catalogStore2 = mock(CatalogStore.class);
-        catalogStore3 = mock(CatalogStore.class);
-        catalogStoreMap.put("destination1", catalogStore1);
-        catalogStoreMap.put("destination2", catalogStore2);
-        catalogStoreMap.put("destination3", catalogStore3);
-
-        publishedPlaces = new ArrayList<>();
-        publishedPlaces.add("destination1");
-        publishedPlaces.add("destination3");
-
-        destinations = new ArrayList<>();
-        destinations.add("destination1");
-        destinations.add("destination2");
 
         metacard1 = new MetacardImpl();
-        metacard1.setAttribute(new AttributeImpl(PUBLISHED_LOCATIONS, publishedPlaces));
-        metacard1.setAttribute(new AttributeImpl(REGISTRY_ID, "registry1"));
         results = new ArrayList<>();
         results.add(new ResultImpl(metacard1));
 
-        filterBuilder = new GeotoolsFilterBuilder();
+        poller = new AdminPollerTest().new MockedAdminPoller(null);
 
-        poller = new AdminPollerTest().new MockedAdminPoller(null,
-                catalogFramework,
-                filterBuilder,
-                catalogStoreMap);
+        ArrayList<ActionProvider> operationActions = new ArrayList<>();
```

Comment:
We could use Guava's `ImmutableList.of()` or `Arrays.asList()` to create those lists.